src/scss/components/_buttons.scss

```diff
@@ -27,6 +27,50 @@
   }
 }
 
+.usa-button--unstyled {
+  position: relative;
+  text-underline-offset: 3px;
+
+  // In USWDS, unstyled button styles are applied last to take precedence over base button styles.
+  // Since we override base button styles, we must re-apply the unstyled button styles.
+  &.usa-button {
+    @include button-unstyled;
+  }
+
+  // Hover colors are already applied by USWDS via `typeset-link`, but this does not include the
+  // button modifier classes.
+  &:hover,
+  &.usa-button--hover {
+    color: color($theme-link-hover-color);
+  }
+
+  &:active,
+  &.usa-button--active {
+    color: color($theme-link-active-color);
+  }
+
+  &:disabled {
+    color: color('disabled');
+  }
+
+  &.usa-button:not([disabled]):focus,
+  &.usa-button:not([disabled]).usa-focus {
```

Comment:
Just to clarify, this is different from the stanza as below because the `&` means these styles *when they apply to `.usa-button--unstyled`* right, and the one below is when they're on any button?

Reply:
> Just to clarify, this is different from the stanza as below because the `&` means these styles _when they apply to `.usa-button--unstyled`_ right, and the one below is when they're on any button?

Yeah, when compiled it's the difference of:

```css
.usa-button--unstyled.usa-button:not([disabled]):focus {
  /* ... these styles */
}
```

vs.

```css
.usa-button:not([disabled]).usa-focus {
  /* ... default focus styles */
}
```

Where the intention with these styles is to unset and replace the default focus styles, only for the unstyled buttons. _Ideally_ we could just use the default styles, but it was surprisingly difficult to reconcile the focus style box shadow + border radius, hence recreating it with a positioned pseudo-element.